### DISCUSSION TITLE
Is it possibile to use a custom layout pattern in file based routing?

### DISCUSSION THREAD

**Discussion by mattiaz9** [TanStack/router#2218] [Q&A]
Personally, I dislike the underscore `_...` pattern for layouts and I wonder if it's possible to customize it.
I see in the options there is a `routeFileIgnorePattern` but nothing else.

I would like to use something more like this `<layout_name>`, similar to how groups are handled `(group_name)`.

I think that would be cleaner and more readable (just personal preference).

**Comment by schiller-manuel** [comment]
no, this is currently not possible. could potentially be added in future.
